Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 55;
```

Comment:
Should this be reverted?

Reply:
Xcode does its thing, someone touched it with a version before Xcode 16. This is fine.